packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/recent.tsx

```diff
@@ -29,7 +29,7 @@ const RecentComponent = ({
         toggleStrategies(true);
         setTimeout(() => {
             getSavedWorkspaces().then(recent_strategies => setDashboardStrategies(recent_strategies));
-        }, 200);
+        }, 100);
```

Comment:
Can this be handled in `useEffect`?

Reply:
removed setTimeout and handled with async await now